fdbserver/workloads/ReadHotDetection.actor.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * ReadHotDetection.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
```

Comment:
copyright year to 2020?

Reply:
Good catch

Reply:
I think there are lots of files that need to update their copyright header to 2020. I will leave this one to be changed along with others.